templates/index.html.j2

```diff
@@ -1,8 +1,8 @@
 <html>
-    <title>{Blue/Green} Application</title>
+    <title>Blue/green Application</title>
```

Comment:
You should use a variable to define the name of the application. It should be either `Blue` or `Green` not both.